projects/angular/src/forms/datepicker/_datepicker.clarity.scss

```diff
@@ -8,40 +8,32 @@
 @use '../../utils/mixins';
 @use '../../utils/variables/variables.global';
 @use 'variables.datepicker' as datepicker-variables;
+@use '@cds/core/tokens/tokens.scss';
+
 @include meta.load-css('properties.datepicker');
 
 @mixin generate-calendar-button(
-  $dimension: clr-calendar-cell-width-height,
-  $height-fallback: datepicker-variables.$clr-calendar-click-target,
+  $height: datepicker-variables.$clr-calendar-click-target,
   $padding: datepicker-variables.$clr-calendar-btn-padding
 ) {
   //Dimensions
-  @include mixins.min-equilateral-var($dimension, $height-fallback, variables.$clr-use-custom-properties);
+  @include mixins.min-equilateral($height);
   padding: 0 $padding;
   margin: 0 !important; //Because stupid forms target buttons directly. We really need to refactor forms.
 
   //Other Props
   color: inherit;
   border: none;
-  @include mixins.css-var(
-    border-radius,
-    clr-calendar-cell-border-radius,
-    variables.$clr-global-borderradius,
-    variables.$clr-use-custom-properties
-  );
+  border-radius: tokens.$cds-alias-object-border-radius-100;
```

Comment:
Ok, why do we decide to directly bring a CDS variable here and not keep a configurable `clr-calendar-cell-border-radius` property and variable that point to the CDS var?

What's the difference between this case and the others where we do the latter?

Reply:
`clr-calendar-cell-border-radius` is introduced with TM PR #878. The property is not needed any more and using CDS token directly is cleaner.

Reply:
I still think this can be exposed, but I can agree with the "when in doubt, take the conservative approach". We can add it back if needed.